content/organisation.md

```diff
@@ -0,0 +1,32 @@
+---
+title: "Organisation"
+date: 2022-02-01T11:53:42Z
+draft: false
+---
+<center> {{< figure src="/images/swifthep-organisation.png" alt="swift-hep Organisation" width="600px">}} </center>
+<h3> Management </h3>
+The SWIFT-HEP coordinator is <a href="mailto:davide.costanzo@cern.ch"> Davide Costanzo </a> (Sheffield) 
+and deputy <a href="mailto:conor.fitzpatrick@cern.ch ">Conor Fitzpatrick</a> (Manchester). <br>
+The current conveners of the work packages are:
+<ul>
+<li> WP1: <a href="mailto:d.colling@imperial.ac.uk"> David Colling </a> (Imperial, CMS) and <a href="alastair.dewhurst@stfc.ac.uk"> Alastair Dewhurst </a> (RAL-SCD, Tier0)
+<li> WP2: Andy Buckley (Glasgow, ATLAS) and <a href="mailto:marek.schoenherr@cern.ch"> Marek Schoenherr </a> (Durham, Theory) 
```

Comment:
Can't speak for Marek, but since it is supported, why not `Sch&ouml;nherr` (renders as Sch&ouml;nherr)?